crates/symbolicator-sources/src/paths.rs

```diff
@@ -463,9 +463,7 @@ pub fn get_directory_paths(
 
 /// Parses a symstore path into a possible [`FileType`] and an [`ObjectId`].
 pub fn parse_symstore_path(path: &str) -> Option<(&'static [FileType], ObjectId)> {
-    let mut split = path.splitn(4, '/');
-    // Skip the leading / that the path contains
-    split.next()?;
+    let mut split = path.splitn(3, '/');
```

Comment:
This assumes that the path _never_ starts with a `/`, right? Is that how it works in practice?

Reply:
It is, yes.
All the testcases btw I took literally from https://github.com/dotnet/symstore/blob/main/docs/specs/SSQP_Key_Conventions.md